docs/authorization-configuration.md

```diff
@@ -25,7 +25,8 @@ auth: {
     scope: 'audience:server:client_id:client openid profile email groups',
     redirect_uri: '',
     post_logout_redirect_uri: '/logout.html',
-    automaticSilentRenew: true
+    automaticSilentRenew: true,
+    userInfoFn:()=>{}
```

Comment:
As a user, I'd like to get token info here, same the provider receives. `const tokenInfo = this.parseIdToken(authData.idToken);`  But watch out, OIDC provider does not have `getAuthData` function yet.
Might be enough to hand over `JSON.parse(localStorage.getItem('luigi.auth'))`.